### ISSUE TITLE
love.math.colorFromBytes missing

### ISSUE THREAD

**Issue by farism** [hazzard993/love-typescript-definitions#39]
I see that `colorToBytes` is declared, but not `colorFromBytes`

**Comment by hazzard993** [comment]
Thanks for reporting that!

I'll go through and re-check the functions.

If this is stopping you at the moment you can add this code within a `.ts` or `.d.ts` file in your project to use this function until the next update.

```ts
declare module "love.math" {
    type RGBA = [number, number, number, number?];

    /**
     * Converts a color from 0..255 to 0..1 range.
     *
     * @param rgba The red, green, blue and alpha (optional) values within a 0..255 range.
     * @returns All given values now within a 0..1 range.
     * @tupleReturn
     * @link [love.math.colorFromBytes](https://www.love2d.org/wiki/love.math.colorFromBytes)
     */
    export function colorFromBytes<T extends RGBA>(...rgba: T): T;
}
```

```ts
const [r, g, b, a] = love.math.colorFromBytes(255, 255, 255, 255);
```

**Comment by farism** [comment]
Thanks!

**Comment by hazzard993** [comment]
Fixed in #40 in love-typescript-definitions@0.19.0
